src/forms/yup.js

```diff
@@ -24,7 +24,11 @@ const ARRAY_INDEX_REGEX = /\[([^)]+)\]/;
 yup.setLocale({
   mixed: {
     default: params => ({ key: 'form.validation_field_invalid', params }),
-    required: params => ({ key: 'form.validation_field_required', params }),
+    required: params => {
+      return params.path === 'location'
```

Comment:
question: does this only work for the field named `location`? We want it to work for any <select>.

Reply:
Just added a custom validation rule just for selected that can be added to any validation schema